src/client/src/database.rs

```diff
@@ -215,6 +215,7 @@ impl Database {
                 start: start.to_string(),
                 end: end.to_string(),
                 step: step.to_string(),
+                lookback: "".to_string(),
```

Comment:
Not sure about this line: use default or update the `prom_range_query` signature to pass this parameter. If the latter - will it be a breaking change/require docs update etc?

Reply:
We can use default here as most users won't customize the `lookback` parameter.

Reply:
Great! Thanks for confirming, I'll put `DEFAULT_LOOKBACK_STRING` then